src/index.js

```diff
@@ -64,6 +63,38 @@ function displaytask() {
       }
       localStorage.setItem('baseData', JSON.stringify(dataBase));
     });
+
+    span.addEventListener('change', () => {
+      const index = i;
+      dataBase.forEach((newInput, i) => {
+        if (index === i) {
+          newInput.description = span.value;
+        }
+      });
+
+      localStorage.setItem('baseData', JSON.stringify(dataBase));
+    });
+
+    checkbox.addEventListener('change', () => {
+      if (checkbox.checked) {
+        dataBase[i].completed = true;
+      } else {
+        dataBase[i].completed = false;
+      }
+      localStorage.setItem('baseData', JSON.stringify(dataBase));
+    });
+
+    clearButton.addEventListener('click', () => {
+      if (checkbox.checked && true) {
+        dataBase.splice(i, 1);
+        list.remove();
+        for (let i = 0; i < dataBase.length; i += 1) {
+          dataBase[i].index = i + 1;
+        }
+        localStorage.setItem('baseData', JSON.stringify(dataBase));
+      }
+    });
```

Comment:
- [ ] [REQUIRED] Good job so far :+1:  But as the previous reviewer mentioned this functionality works but when I refresh the page the status  `completed: true`   I can still not clear the task , I think you can check also your arrow . Please fix this issue in order to make your todo list work correctly 

> * This functionality works until the page is refreshed. Please make sure the updated array of items is stored in local storage, so that the user gets the updated list of tasks after the page reloads.

Reply:
Hi @jdbs9514, as mentioned by the previous reviewer, this problem still persists. Once a task is marked completed, upon reloading it doesn't display a tick mark beside the task. Also, there is some problem with the way you are updating the populating the array/local storage because when clear all completed tasks, followed by a reload, different tasks get deleted that were not meant to be deleted. Consider fixing this issue.